modules/oe_corporate_countries_address/tests/src/Kernel/AvailableCountriesSubscriberTest.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Drupal\Tests\oe_corporate_countries_address\Kernel;
+
+use Drupal\entity_test\Entity\EntityTest;
+use Drupal\field\Entity\FieldConfig;
+use Drupal\field\Entity\FieldStorageConfig;
+use Drupal\Tests\oe_corporate_countries\Kernel\CorporateCountriesRdfKernelTestBase;
+
+/**
+ * Tests the AvailableCountriesSubscriber class.
+ */
+class AvailableCountriesSubscriberTest extends CorporateCountriesRdfKernelTestBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  protected static $modules = [
+    'user',
+    'field',
+    'entity_test',
+    'address',
+    'oe_corporate_countries',
+    'oe_corporate_countries_address',
+  ];
+
+  /**
+   * Name of the field.
+   *
+   * @var string
+   */
+  protected string $fieldName = 'field_country';
+
+  /**
+   * Name of the field type.
+   *
+   * @var string
+   */
+  protected string $fieldTypeName = 'address_country';
+
+  /**
+   * Name of the entity type.
+   *
+   * @var string
+   */
+  protected string $entityTypeName = 'entity_test';
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function setUp(): void {
+    parent::setUp();
+
+    FieldStorageConfig::create([
+      'field_name' => $this->fieldName,
+      'entity_type' => $this->entityTypeName,
+      'type' => $this->fieldTypeName,
+    ])->save();
+
+    FieldConfig::create([
+      'label' => 'My field',
+      'field_name' => $this->fieldName,
+      'entity_type' => $this->entityTypeName,
+      'bundle' => $this->entityTypeName,
+    ])->save();
+
+  }
+
+  /**
+   * Tests the deprecated countries are removed from the countries array.
+   * */
+  public function testGetAvailableCountriesArray(): void {
+
+    // Get available_countries from field.
+    // This doesn't have the deprecated countries.
+    $entity = EntityTest::create();
+    $availableCountries = $entity->get('field_country')->appendItem()->getAvailableCountries();
```

Comment:
We should be using camel case only in methods and functions names and use snake case for naming variables.
```suggestion
    $actual_countries = $entity->get('field_country')->appendItem()->getAvailableCountries();
```

Reply:
We did that change. Thanks.